cvss/interactive.py

```diff
@@ -73,13 +73,13 @@ def ask_interactively(version=3, all_metrics=False, no_colors=False):
                 name_with_hints = name_with_hints.replace(letter, '(' + letter + ')')
 
             # Exceptions for hints
-            if version == 3 and name_with_hints == 'Not Defined':
+            if version >= 3.0 and name_with_hints == 'Not Defined':
```

Comment:
Don't you think that this will be easier to read?

```suggestion
            if version >= 3.0 and name_with_hints == 'Not Defined':
                name_with_hints = '(X)Not Defined'
            elif version < 3.0 and name_with_hints == '(P)roof of (C)oncept':
                name_with_hints = '(P)roof (O)f (C)oncept'
            elif version < 3.0 and name_with_hints == '(U)nconfirmed':
                name_with_hints = '(U)n(C)onfirmed'
            elif version < 3.0 and name_with_hints == '(U)ncorroborated':
                name_with_hints = '(U)nco(R)roborated'
```

or even:

```suggestion
            if version >= 3.0 and name_with_hints == 'Not Defined':
                name_with_hints = '(X)Not Defined'
            elif version < 3.0:
                name_with_hints = {
                    '(P)roof of (C)oncept': '(P)roof (O)f (C)oncept',
                    '(U)nconfirmed': '(U)n(C)onfirmed',
                    '(U)ncorroborated': '(U)nco(R)roborated',
                }.get(name_with_hints, name_with_hints)
```

Reply:
I like it! Thanks!